src/Sylius/Bundle/UiBundle/Twig/UndefinedCallableHandler.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UiBundle\Twig;
+
+use Twig\Error\SyntaxError;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+final class UndefinedCallableHandler
+{
+    /**
+     * @var string[]
+     */
+    private const FUNCTION_COMPONENTS = [
+        'sm_can' => 'winzou/state-machine-bundle',
+        'sylius_grid_render_bulk_action' => 'sylius/grid-bundle',
+        'sylius_grid_render_field' => 'sylius/grid-bundle',
+        'sylius_grid_render_filter' => 'sylius/grid-bundle',
```

Comment:
Why did you list these three dependencies? They should be available
<img width="591" alt="image" src="https://user-images.githubusercontent.com/6213903/206479849-c383bcec-3786-4049-9fbb-64a86cdde56f.png">

Reply:
Hum indeed, that seems to be a mistake. The idea is to continue using UiBundle on Symfony projects. But some of the templates needs other dependencies and we have to inform about missing ones to use them.